c-hash/assignment10/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace assignment10
+{
+    class Program10
+    {
+        public static void Main(string[] args)
+        {
+            int number, palindrome,originalnumber;
+            Console.WriteLine("enter the number to do a palindrome check");
+            number=Convert.ToInt32(Console.ReadLine());
+           originalnumber = number;
+            for(palindrome = 0;number > 0; )
```

Comment:
while loop is the correct candidate for this operation

Reply:
ok